components/common/InputField/InputField.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, InputHTMLAttributes, forwardRef } from "react";
+import styles from "./inputField.module.css";
+import EyeOff from "@/images/ic_eye-off.svg";
+import EyeOn from "@/images/ic_eye-on.svg";
+
+type Props = {
+  id: "email" | "password" | "passwordConfirm";
+  type: "text" | "email" | "password";
+  label: "이메일" | "비밀번호" | "비밀번호 확인";
+  error?: string;
+} & InputHTMLAttributes<HTMLInputElement>;
+
+const InputField = forwardRef<HTMLInputElement, Props>(
+  ({ id, type, label, error, ...rest }, ref) => {
+    const [showPassword, setShowPassword] = useState(false);
+
+    return (
+      <div className={styles.section}>
```

Comment:
props로 className을 받아서 이렇게 적용해줘도 좋을 것 같아요!
```suggestion
      <div className={cx(styles.section, className)}>
```